docs/schema-update-playbook.md

```diff
@@ -0,0 +1,25 @@
+# Bytebase Schema Update Playbook
```

Comment:
Schema Update Guide

be consistent with other doc

Reply:
Done.